dissect/target/plugins/os/windows/recyclebin.py

```diff
@@ -125,6 +129,44 @@ def read_bin_file(self, bin_path: TargetPath) -> RecycleBinRecord:
             _user=user,
         )
 
+    def read_recycle_deleted_folder(self, folder_path: TargetPath) -> RecycleBinRecord:
```

Comment:
```suggestion
    def read_recycle_deleted_folder(self, path: TargetPath) -> RecycleBinRecord:
```
And rename accordingly.